src/plugins/kernel_install/utils.sh

```diff
@@ -40,7 +40,7 @@ function cmd_manager()
       ;;
   esac
 
-  eval "$@"
+  "$@"
```

Comment:
You dropped the eval here and re-introduced it in the next commit; in this sense, just remove this change from this commit.